ai/anthropic/v0/config/tasks.json

```diff
@@ -227,7 +227,7 @@
             "reference",
             "template"
           ],
-          "title": "System message",
+          "title": "System Message",
```

Comment:
nit: we should either document that titles **must** be title-cased in the contribution guidelines, or use the `titleCaseWithArticles` function [in the template](https://github.com/instill-ai/component/blob/main/tools/compogen/pkg/gen/readme.go#L164) to unify the titles across the documents. If we don't care particularly about the JSON definition, I'd go with the second option.